daiquiri/files/views.py

```diff
@@ -36,3 +45,27 @@ def get(self, request, file_path, **kwargs):
             return render_with_layout(request, file_path)
         else:
             return send_file(request, file_path)
+
+
+class SearchView(View):
+
+    root = None
+
+    def get(self, request, **kwargs):
+
+        search_string = request.GET.get("q", "")
+
+        results = Searcher.search_for_string(string_query=search_string)
+
+        paginator = Paginator(results, 5)
```

Comment:
make the number of results in the search page to be in SETTINGS.

Reply:
Done!